content/ngf/reference/permissions.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Permissions
+description: NGINX Gateway Fabric permissions required by components.
+weight: 300
+toc: true
+type: reference
+product: NGF
+---
+
+## Overview
+
+NGINX Gateway Fabric uses a split-plane architecture with three components that require different permissions:
+
+- **Control Plane**: Manages Kubernetes APIs and data plane deployments. Needs broad API access but handles no user traffic.
+- **Data Plane**: Processes user traffic. Requires minimal permissions since configuration comes from control plane via secure gRPC.
+- **Certificate Generator**: One-time job that creates TLS certificates for inter-plane communication.
+
+## Security Context
+
+All components share these security settings:
+
+- **User ID**: 101 (non-root)
+- **Group ID**: 1001  
+- **Capabilities**: All dropped (`drop: ALL`)
+- **Root Filesystem**: Read-only
+- **Seccomp**: Runtime default profile
+
+## Control Plane
+
+Runs as a single container in the `nginx-gateway` deployment.
+
+**Additional Security Settings:**
+- **Privilege Escalation**: Disabled (may need enabling for NGINX reload in some environments)
+
+**Volumes:**
+- `nginx-agent-tls` (Secret) - TLS certificates for control plane communication
+
+**RBAC Permissions:**
+- **Secrets, ConfigMaps, Services**: Create, update, delete, list, get, watch
+- **Deployments, DaemonSets**: Create, update, delete, list, get, watch
+- **ServiceAccounts**: Create, update, delete, list, get, watch
+- **Namespaces, Pods**: Get, list, watch
+- **Events**: Create, patch
+- **EndpointSlices**: List, watch
+- **Gateway API resources**: List, watch (read-only) + update status subresources only
+- **NGF Custom resources**: Get, list, watch (read-only) + update status subresources only
+- **Leases**: Create, get, update (for leader election)
+- **CustomResourceDefinitions**: List, watch
+- **TokenReviews**: Create (for authentication)
+
+## Data Plane
+
+NGINX containers managed by the control plane. No RBAC permissions needed since configuration comes via secure gRPC.
+
+**Additional Security Settings:**
+- **Privilege Escalation**: Disabled
+- **Sysctl**: `net.ipv4.ip_unprivileged_port_start=0` (enables binding to ports < 1024)
+
+**Volumes:**
+
+| Volume | Type | Purpose |
```

Comment:
To avoid having to update these as they change over time, I would be more general, and say that there are a variety of emptyDir volumes used for creating write-able directories for nginx config. As well as a few Secret mounts for storing certificates for TLS communication with the control plane, and mounting the JWT for running NGINX Plus. And the projected token as well.